<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetAbout</class>
 <widget class="QWidget" name="WidgetAbout">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>177</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>346</width>
    <height>177</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>346</width>
    <height>177</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: rgb(50, 50, 52);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>346</width>
       <height>57</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
background-color: rgb(50, 50, 52);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/resources/AVSoft2.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QToolButton" name="btnClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton 
{
 		 	image: url(:/resources/Cancel gray.png);
           icon-size: 12px;
           padding-left: 10px;
           padding-right: 10px;
           padding-top: 5px;
           padding-bottom: 5px;
        
			border-radius:5px;
}

QToolButton:hover
 {
	image: url(:/resources/White gray.png);
}

QToolButton[style=&quot;light&quot;] 
{
 		 	
			image: url(:/resources/CloseBlack.png);
           icon-size: 12px;
           padding-left: 10px;
           padding-right: 10px;
           padding-top: 5px;
           padding-bottom: 5px;
        
			border-radius:5px;
}

QToolButton:hover[style=&quot;light&quot;] 
 {
	
	image: url(:/resources/Cancel gray.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>346</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="agree">
        <property name="maximumSize">
         <size>
          <width>235</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	font: 400 bold  9pt &quot;Roboto&quot;;
	color:rgb(255, 102, 51);
}</string>
        </property>
        <property name="text">
         <string>Соглашение с конечным пользователем:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="agreeButton">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
font: 400 bold  9pt &quot;Roboto&quot;;
color:rgb(255, 102, 51);
border:none;
background-color: transparent;
}

QPushButton:hover
{
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="about">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	font: 400 bold  9pt &quot;Roboto&quot;;
	color:rgb(255, 102, 51);
}</string>
        </property>
        <property name="text">
         <string>Справка: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="aboutButton">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
font: 400 bold  9pt &quot;Roboto&quot;;
color:rgb(255, 102, 51);
border:none;
background-color: transparent;
}

QPushButton:hover
{
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
